DESCRIPTION

```diff
@@ -20,7 +20,8 @@ Imports:
     htmlwidgets,
     png,
     webshot,
-    lifecycle
+    lifecycle,
+    V8
```

Comment:
Don't you mean to add `V8` as a suggest? Specially since V8 is heavy might get tricky to get installed in some platforms due to system dependencies. If anything, I think it makes more sense to retrieve `V8` dynamically, similar to `pins` https://github.com/rstudio/pins/blob/master/R/board_rsconnect.R#L1-L5 -- This would put the burden of installing `V8` only to users that really need this functionality.

Reply:
Thanks for pointing this out.  It's a great point and I have changed the dependency on `V8` to `Suggests`